packages/core/src/lib/notifire.ts

```diff
@@ -51,13 +51,14 @@ export class Notifire extends EventEmitter {
   }
 
   async registerProvider(
+    providerId: string,
```

Comment:
I would probably make this one optional, and use the internalId as the default if it was not provided. Similar to "passport-js" approach when registering providers.